openraft/tests/fixtures/mod.rs

```diff
@@ -535,6 +552,13 @@ where
         nodes.remove(&id);
     }
 
+    /// Unblock the network of the specified node.
+    #[tracing::instrument(level = "debug", skip(self))]
+    pub fn unblock_node(&self, id: C::NodeId) {
+        let mut nodes = self.unreachable_nodes.lock().unwrap();
+        nodes.remove(&id);
+    }
```

Comment:
It'd be better to merge block_node and unblock_node into one method. e.g. `enable_connect(node_id, enabled: bool)`.

Reply:
That's the name I'm look for!